mlir/lib/Dialect/Vector/Transforms/VectorTransforms.cpp

```diff
@@ -1795,6 +1795,80 @@ struct BreakDownVectorReduction final : OpRewritePattern<vector::ReductionOp> {
   unsigned maxNumElementsToExtract = 0;
 };
 
+/// Pattern aiming to fold a series of ops mulf(tr(broadcast(A)), broadcast(B))
+/// into vector.outerproduct(A, B) such as :
```

Comment:
[nit] Keep it shorter. In particular, "series" might suggest that more than one occurance of `mulf(tr(broadcast(A)), broadcast(B))` is required:
```suggestion
/// Fold `mulf(tr(broadcast(A)), broadcast(B))` into `vector.outerproduct(A, B)`. Example:
```